_summaries/kingma2014adam_1.md

```diff
@@ -3,17 +3,39 @@ layout: summary
 title: Summary
 paper: {{paper_tag}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: jordi1215
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+Adam (derived from adaptive moment estimation) is an optimization algorithm that can be used instead of the classical stochastic gradient descent procedure to update network weights iterative based in training data.
+
+
+![Adam algorithm](Kingma2014adam_1_a.png)
+
+* Straightforward to implement.
+* Computationally efficient.
+* Little memory requirements.
+* Invariant to diagonal rescale of the gradients.
+* Well suited for problems that are large in terms of data and/or parameters.
+* Appropriate for non-stationary objectives.
+* Appropriate for problems with very noisy/or sparse gradients.
+* Hyper-parameters have intuitive interpretation and typically require little tuning.
+
+The authors describe Adam as combining the advantages of two other extensions of stochastic gradient descent. Specifically:
+
+* Adaptive Gradient Algorithm (AdaGrad) - that maintains a per-parameter learning rate that improves performance on problems with sparse gradients (e.g. natural language and computer vision problems).
+
+* Root Mean Square Propagation (RMSProp) - that also maintains per-parameter learning rates that are adapted based on the average of recent magnitudes of the gradients for the weight (e.g. how quickly it is changing). This means the algorithm does well on online and non-stationary problems (e.g. noisy).
+Adam realizes the benefits of both AdaGrad and RMSProp.
+
+Instead of adapting the parameter learning rates based on the average first moment (the mean) as in RMSProp, Adam also makes use of the average of the second moments of the gradients (the uncentered variance).
+
+Specifically, the algorithm calculates an exponential moving average of the gradient and the squared gradient, and the parameters beta1 and beta2 control the decay rates of these moving averages.
+
+The initial value of the moving averages and beta1 and beta2 values close to 1.0 (recommended) result in a bias of moment estimates towards zero. This bias is overcome by first calculating the biased estimates before then calculating bias-corrected estimates.
```

Comment:
The initial value part might be too detailed for the reader. Focusing on high-level ideas helps improve readability.